### PR TITLE
KOGITO 4541: Move quickstart code as extended usecases to current springboot examples

### PR DESCRIPTION
**Pull request by nmirasch**
Jira: https://issues.redhat.com/browse/KOGITO-4541

Added two springboot examples showing consoles interaction:
- process-usertasks-springboot-with-console
- process-usertasks-with-security-oidc-springboot-with-console

### CODE REVIEW

File: process-usertasks-springboot-with-console/docker-compose/startServices.sh [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+rm -rf $PERSISTENCE_FOLDER
+
+mkdir -p $PERSISTENCE_FOLDER
+
+if [ -d "$KOGITO_EXAMPLE_PERSISTENCE" ]
+then
+    cp $KOGITO_EXAMPLE_PERSISTENCE/*.proto $PERSISTENCE_FOLDER/
+else
+    echo "$KOGITO_EXAMPLE_PERSISTENCE does not exist. Have you compiled process-usertask-quarkus-extended project?"
```
Comment (cristianonicolai) [inline process-usertasks-springboot-with-console/docker-compose/startServices.sh]:
```suggestion
    echo "$KOGITO_EXAMPLE_PERSISTENCE does not exist. Have you compiled process-usertasks-springboot-with-console project?"
```

File: process-usertasks-springboot-with-console/docker-compose/startServices.sh [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+KOGITO_HIRING_SVG_FOLDER=../target/classes/META-INF/processSVG
+
+mkdir -p $SVG_FOLDER
+
+if [ -d "$KOGITO_HIRING_SVG_FOLDER" ]
+then
+    cp $KOGITO_HIRING_SVG_FOLDER/*.svg $SVG_FOLDER
+else
+    echo "$KOGITO_HIRING_SVG_FOLDER does not exist. Have you compiled process-usertask-quarkus-extended project?"
```
Comment (cristianonicolai) [inline process-usertasks-springboot-with-console/docker-compose/startServices.sh]:
```suggestion
    echo "$KOGITO_HIRING_SVG_FOLDER does not exist. Have you compiled process-usertasks-springboot-with-console project?"
```

File: process-usertasks-springboot-with-console/pom.xml [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+      <artifactId>process-management-addon</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.kie.kogito</groupId>
+      <artifactId>process-management-springboot-addon</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.kafka</groupId>
+      <artifactId>spring-kafka</artifactId>
+      <optional>true</optional>
```
Comment (cristianonicolai) [inline process-usertasks-springboot-with-console/pom.xml]: should be in by default?

File: process-usertasks-with-security-oidc-springboot-with-console/mvnw [hunk 4]
```diff
@@ -0,0 +1,310 @@
+#!/bin/sh
```
Comment (cristianonicolai) [inline process-usertasks-with-security-oidc-springboot-with-console/mvnw]: @nmirasch I would avoid adding this file to git, myabe only needed at the root of the project

File: process-usertasks-with-security-oidc-springboot-with-console/mvnw.cmd [hunk 5]
```diff
@@ -0,0 +1,182 @@
+@REM ----------------------------------------------------------------------------
```
Comment (cristianonicolai) [inline process-usertasks-with-security-oidc-springboot-with-console/mvnw.cmd]: same

File: process-usertasks-with-security-oidc-springboot-with-console/pom.xml [hunk 6]
```diff
@@ -0,0 +1,158 @@
...
+      <artifactId>process-management-addon</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.kie.kogito</groupId>
+      <artifactId>process-management-springboot-addon</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.kafka</groupId>
+      <artifactId>spring-kafka</artifactId>
+      <optional>true</optional>
```
Comment (cristianonicolai) [inline process-usertasks-with-security-oidc-springboot-with-console/pom.xml]: same regarding optional

**Comment by nmirasch** [comment]
@cristianonicolai @radtriste updated, could you take a look? Thanks!

File: process-usertasks-with-security-oidc-springboot-with-console/mvnw (hunk 4 above)
Comment (nmirasch) [reply process-usertasks-with-security-oidc-springboot-with-console/mvnw]: removed

File: process-usertasks-with-security-oidc-springboot-with-console/mvnw.cmd (hunk 5 above)
Comment (nmirasch) [reply process-usertasks-with-security-oidc-springboot-with-console/mvnw.cmd]: removed

File: process-usertasks-springboot-with-console/pom.xml (hunk 3 above)
Comment (nmirasch) [reply process-usertasks-springboot-with-console/pom.xml]: I agree, thanks!

File: process-usertasks-with-security-oidc-springboot-with-console/pom.xml (hunk 6 above)
Comment (nmirasch) [reply process-usertasks-with-security-oidc-springboot-with-console/pom.xml]: @cristianonicolai I removed the optional from spring-kafka in both examples. They are mandatory in both, I guess this makes sense when we expose different profiles, but in this case we don't have them.

**Review by radtriste — APPROVED** [review]

**Review by cristianonicolai — APPROVED** [review]

**Pull request merged**